cmd/basin/publication.go

```diff
@@ -247,6 +249,12 @@ func newPublicationUploadCommand() *cli.Command {
 				Destination: &secure,
 				Value:       true,
 			},
+			&cli.StringFlag{
+				Name:        "timestamp",
+				Usage:       "The time the file was created",
```

Comment:
should we explain here that the default is "now" according to client time? or is that obvious in the help output?

Reply:
yeah not very obvious. i can add more info. but it's actually the "now" [according to UTC](https://github.com/tablelandnetwork/basin-cli/pull/16/files#diff-73777451ff3bdcfcd6ee6970408801e8731a6368a183dfca6d1e1b7df2195fc7R256)

Reply:
cool. i meant "now" as UTC according to client's clock, but maybe that's obvious

Reply:
funny, didn't realize the help would show the calculated value

```
OPTIONS:
   --private-key value  Ethereum wallet private key
   --name value         Publication name
   --secure             Uses TLS connection (default: true)
   --timestamp value    The time the file was created (default: "1699641318")
   --help, -h           show help
```

quite weird because every time you run, shows a different value

Reply:
ha, that is funny. maybe okay if you just saying something like "The time the file was created (defaults to now UTC)